src/vs/editor/standalone/browser/standalone-tokens.css

```diff
@@ -4,9 +4,10 @@
  *--------------------------------------------------------------------------------------------*/
 
 
-/* Default standalone editor font */
+/* Default standalone editor fonts */
 .monaco-editor {
-	font-family: system-ui, -apple-system, BlinkMacSystemFont, "Segoe WPC", "Segoe UI", "HelveticaNeue-Light", "Ubuntu", "Droid Sans", sans-serif;
+	font-family: -apple-system, BlinkMacSystemFont, "Segoe WPC", "Segoe UI", "HelveticaNeue-Light", system-ui, "Ubuntu", "Droid Sans", sans-serif;
+	--monaco-monospace-font: "SF Mono", Monaco, Menlo, Consolas, "Ubuntu Mono", "Liberation Mono", "DejaVu Sans Mono", "Courier New", monospace;
```

Comment:
@alexdima fyi this defines the `monaco-monospace-font` for standalone editor to be a union of the 3 platform specific rules we have in the workbench

Reply:
:+1: Thank you!